### PR TITLE
Add examples for NLP in pipeline

### PR DESCRIPTION
**Pull request by novaturient95**
# PR into Azure/azureml-examples

## Checklist

I have:

- [x] read and followed the contributing guidelines

## Changes

- added examples for NLP tasks in pipeline

fixes #

### CODE REVIEW

File: cli/jobs/pipelines/automl/text-classification-multilabel-paper-categorization-pipeline.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      max_trials: 1
+      timeout_minutes: 60
+    target_column_name: terms
+    training_data: ${{parent.inputs.text_classification_multilabel_training_data}}
+    validation_data: ${{parent.inputs.text_classification_multilabel_validation_data}}
+    # currently need to specify outputs "mlflow_model" explicitly to reference it in following nodes
+    outputs:
+      best_model:
+        type: mlflow_model
+  show_output:
```
Comment (wangchao1230) [inline cli/jobs/pipelines/automl/text-classification-multilabel-paper-categorization-pipeline.yml:37]: what's typical user next step when thy have an automl outputed model?
Maybe we can make the example not so dummy

File: sdk/jobs/pipelines/1h_automl_in_pipeline/automl-text-classification-multilabel-in-pipeline.ipynb [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# import required libraries\n",
+    "from azure.identity import DefaultAzureCredential, InteractiveBrowserCredential\n",
+    "\n",
+    "from azure.ml import MLClient, dsl, Input, command, Output\n",
```
Comment (wangchao1230) [inline sdk/jobs/pipelines/1h_automl_in_pipeline/automl-text-classification-multilabel-in-pipeline.ipynb:42]: need to update to azure.ai.ml

File: sdk/jobs/pipelines/1h_automl_in_pipeline/automl-text-classification-multilabel-in-pipeline.ipynb [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Define pipeline\n",
+    "@dsl.pipeline(\n",
```
Comment (wangchao1230) [inline sdk/jobs/pipelines/1h_automl_in_pipeline/automl-text-classification-multilabel-in-pipeline.ipynb:122]: we recommend to use @pipeline from our latest PM requirements.

**Review by CESARDELATORRE — APPROVED** [review]
LGTM, I'll need to spend more time trying these PIPELINE examples with a compatible wheel, though. :)

**Pull request merged**